app/controllers/application_controller.rb

```diff
@@ -13,4 +15,9 @@ def switch_locale(&action)
   def unauthorized
     redirect_to root_url, alert: t('.no_access')
   end
+
+  def after_sign_in_path_for(user)
```

Comment:
What this method do? Where do you use it? I think it is not use alt all , so you should remove it.